### DISCUSSION TITLE
Question: What does "pen color" do?

### DISCUSSION THREAD

**Discussion by VOOM108** [LibreHardwareMonitor/LibreHardwareMonitor#1100] [Q&A]
I changed pen color on a few values and nothing happened. What does that do? Is there a manual somewhere for things like this?  

If it is a color used in the plot, then that may be why: the plot is always empty, and I don't know how to add data to it.

**Comment by RealSourceOfficial** [comment]
1. The pen color affects the color of the device on the graph.
2. Click the checkmark next to a device to display it on the plot.
